### PR TITLE
Convert all gifs to mp4

### PR DESCRIPTION
**Pull request by logason**
We should stop using `.gif` in the docs and use `.mp4` instead. They are more efficient and better for performance ✨ 

Here's a before and after for the data design deep dive doc:

Before, with GIFs:
Screenshot 2021-07-21 at 16 21 49

After, with MP4s:
Screenshot 2021-07-21 at 16 21 45

### CODE REVIEW

**Comment by hugihlynsson** [comment]
#### 📝Changed routes:


| [**…gn/day-to-day-workflow**](https://docs-84i270gt0.avo-dev.app/data-design/day-to-day-workflow) | [**…-events-and-properties**](https://docs-84i270gt0.avo-dev.app/data-design/defining-descriptive-events-and-properties) |
|:-:|:-:|
|<a href="https://docs-84i270gt0.avo-dev.app/data-design/day-to-day-workflow">Screenshot of /data-design/day-to-day-workflow</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/QAaUmwebIivHHgAdsu0sKZRciHrcH4ARJpaVleyhrKfRwuW29C3aRa3KwC01d4vpc-x0rjLTw2YBdZ4cfN6maw.png">(view full size)</a> |<a href="https://docs-84i270gt0.avo-dev.app/data-design/defining-descriptive-events-and-properties">Screenshot of /data-design/defining-descriptive-events-and-properties</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/LYg3ZO0tYTqmKqA7FYicaUamjVjvm0fS9PglW3j4MBUEhJHZ0l-ftRxSYFc7OPjIMiWCpcxrjaMMaajsVB_90A.png">(view full size)</a> |



| [**…urpose-meetings-in-avo**](https://docs-84i270gt0.avo-dev.app/data-design/documenting-purpose-meetings-in-avo) | [**…sign/start-data-design**](https://docs-84i270gt0.avo-dev.app/data-design/start-data-design) |
|:-:|:-:|
|<a href="https://docs-84i270gt0.avo-dev.app/data-design/documenting-purpose-meetings-in-avo">Screenshot of /data-design/documenting-purpose-meetings-in-avo</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/AxuNDu1l7Mthv6XEUd5GpVH1gsdrkzN77TCLoi8beVWOial2LN1zJclSyO811Hwlg6CTDg314OZWEu3tdB9pFQ.png">(view full size)</a> |<a href="https://docs-84i270gt0.avo-dev.app/data-design/start-data-design">Screenshot of /data-design/start-data-design</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/6BTs3N2WxaEPIWOdXhzgAAX0uRmXtzKcfTXawMaMStA0kKfefFQLpMqxUjrQA9K4VOaz8MZrd4vEp8zAgxpEbQ.png">(view full size)</a> |



| [**…using-visual-debuggers**](https://docs-84i270gt0.avo-dev.app/explore-tracking-plan/start-using-visual-debuggers) | [**/workspace/branches**](https://docs-84i270gt0.avo-dev.app/workspace/branches) |
|:-:|:-:|
|<a href="https://docs-84i270gt0.avo-dev.app/explore-tracking-plan/start-using-visual-debuggers">Screenshot of /explore-tracking-plan/start-using-visual-debuggers</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/chfAGbywvZk7_rDbMEqAVObyH5PzuScQgQJyiCXpG5qpeJJlVAStciJZBhzC_7MXUbFC00JRhNO_DsfUlNdO1Q.png">(view full size)</a> |<a href="https://docs-84i270gt0.avo-dev.app/workspace/branches">Screenshot of /workspace/branches</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/iquLvTawBA5zLjA-UZo87P2-AnO7cGG66JiptYcFiZpoUjGkoAk-6Xy86KuZ4d320uLQunkWLwTKcH2c9ynpNg.png">(view full size)</a> |



And 1 other route:
- [**/workspace/members**](https://docs-84i270gt0.avo-dev.app/workspace/members)



Commit 92bf0d96b7295a49507df6376b66a302ec218a57 (https://docs-84i270gt0.avo-dev.app).

File: pages/data-design/defining-descriptive-events-and-properties.mdx [hunk 1]
```diff
@@ -23,8 +23,11 @@ selecting or changing event and property names. Note that some analytics tools
 require specific casing for events or properties.
 
 <center>
-  <img
-    src={require('../../images/best-practices/descriptive-events-and-properties/naming-assistant.gif')}
+  <video
+    loop
+    muted
+    autoPlay
```
Comment (bragur) [inline pages/data-design/defining-descriptive-events-and-properties.mdx:29]: I would suggest adding `controls` here as many browsers block autoplaying and in this case the user has to right-click the video (if they realize it's a video at all) and choose Play.

**Review by bragur — CHANGES_REQUESTED** [review]
One video needs `controls`, otherwise amazing!

Screenshot 2021-07-22 at 09 27 04

👀👀👀

File: pages/data-design/defining-descriptive-events-and-properties.mdx (hunk 1 above)
Comment (logason) [reply pages/data-design/defining-descriptive-events-and-properties.mdx:29]: Oops, thought I had put it on all of them but apparently missed this one, fixed in 62eae3d

**Pull request merged**